pkg/mounter/safe_mounter_windows.go

```diff
@@ -71,6 +72,17 @@ func (mounter *CSIProxyMounter) SMBMount(source, target, fsType string, mountOpt
 		}
 	}
 
+	parts := strings.FieldsFunc(source, Split)
+	if len(parts) > 0 && strings.HasSuffix(parts[0], "default.svc.cluster.local") {
+		// replace hostname with IP in the source
+		ip, err := net.ResolveIPAddr("ip4", parts[0])
+		if err != nil {
```

Comment:
if there is error, log a warning msg here, and then continue.
And no matter it resolve IP succeed or not, should always log here.

Reply:
Updated. Please take another look.